src/utils/forecast.js

```diff
@@ -12,10 +12,29 @@ const forecast = (lat, lon, callback) => {
                 description: body.current.weather_descriptions[0],
                 temperature: body.current.temperature,
                 feelslike: body.current.feelslike,
-                icon: body.current.weather_icons[0]
+                icon: body.current.weather_icons[0],
+                location: body.location.region
             })
         }
     })
 }
 
-module.exports.forecast = forecast
\ No newline at end of file
+const forecastErrCheck = (error, { description, temperature, feelslike, icon, location }) => {
+    return new Promise((resolve, reject) => {
+        if (error) {
+            reject(error)
+        }
+        resolve({
```

Comment:
resolve is called even if a promise rejected. Not sure it throw an error, but should be fixed anyway

Reply:
If I'm not mistaken this is a bit from a method that is now deleted.
Please see current geocode, forecast implementation.